Production/python/eostools.py

```diff
@@ -48,16 +45,13 @@ def runEOSCommand(path, cmd, *args):
     I think we should really try and raise an exception in case of problems.
     should be possible as the return code is provided in the tuple returned by runner."""
     
-    lfn = eosToLFN(path)
-    pfn = lfnToPFN(lfn)
-    tokens = cmsIO.splitPFN(pfn)
+    tokens = splitPFN(path)
```

Comment:
There is another issue here (or in general), it calls splitPFN with a `/eos/...` path and not with the protocol prepended, so the regex evaluation above will fail.